src/CircleGraph/CircleViewer.ts

```diff
@@ -103,17 +103,19 @@ export class CircleViewerProvider implements
 
   private _context: vscode.ExtensionContext;
 
-  public static register(context: vscode.ExtensionContext): vscode.Disposable {
```

Comment:
From [#1167](https://github.com/Samsung/ONE-vscode/issues/1167#issue-1329445337),

The function is tidy as is, but I am trying to standardize the registration function as below. 😄 

> Point 1. introduce `public static register(context: vscode.ExtensionContext) : void` function
> Point 2. Register each modules with the static function call
> ```
> OneTreeDataProvider.register(context);
> ```